docker-local/docker-compose.yml

```diff
@@ -280,7 +280,7 @@ services:
 
 ## DFSP A
   dfspa-backend:
-    image: "mojaloop/mojaloop-simulator:v9.4.1"
+    image: "mojaloop/mojaloop-simulator:v0.1.0-pisp"
```

Comment:
I have a feeling that independent image versioning (from `package.json/version`) would bring in future some misunderstanding.